src/base/error_manager.h

```diff
@@ -309,6 +310,13 @@ extern "C"
 }
 #endif
 
+STATIC_INLINE void
+ASSERT_NOT_ERROR (int not_error_code)
+{
+  const int error_code = er_errid ();
+  assert (error_code == NO_ERROR || error_code != not_error_code);
```

Comment:
Is there any case other then (not_error_code == NO_ERROR), for which 
the part  ("error_code == NO_ERROR") is not redundant ?

Reply:
I wanted to make sure that it is not called with `NO_ERROR`. Maybe it should just `assert (not_error_code != NO_ERROR)`.